src/platform/graphics/constants.js

```diff
@@ -1020,9 +1121,22 @@ export const isIntegerPixelFormat = (format) => {
     return pixelFormatInfo.get(format)?.isInt === true;
 };
 
+/**
+ * Returns for the format which can be used to sample the sRGB stored data in linear space, with
+ * automatic sRGB to linear conversion. If it does not exist, the input format is returned. For
+ * example for {@link PIXELFORMAT_RGBA8} the return value is {@link PIXELFORMAT_RGBA8}.
+ *
+ * @param {number} format - The texture format.
+ * @returns {number} The format allowing linear sampling of the texture.
+ * @ignore
+ */
+export const getPixelFormatLinearSampling = (format) => {
```

Comment:
This function takes a linear format and returns its srgb equivalent.

I suggest before this function should be called `mapLinearToGamma` or similar because `getPixelFormatLinearSampling` actually expresses the opposite.